### PR TITLE
changes for PEN-413

### PR DESCRIPTION
**Pull request by mishraomp**

### CODE REVIEW

File: backend/src/components/penRequests.js [hunk 1]
```diff
@@ -489,6 +489,32 @@ async function updateStudentAndDigitalId(req){
...
+    }
+     const params = {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        }
+      };
+    const url = config.get('server:penRequestURL') + `/?pen=${req.query.pen}`;
+    log.silly(`url is ${url}`);
+
+    const response = await axios.get( url, params);
```
Comment (mightycox) [inline backend/src/components/penRequests.js]: This can use utils.getData function

File: frontend/src/components/PenRequestDetail.vue [hunk 2]
```diff
@@ -403,7 +403,10 @@
                     </v-col>
                   </v-row>
                   <v-row justify="end" class="px-3">
-                    <v-col cols="12" xl="3" lg="5" md="5" class="pt-0">
+                    <v-col justify="right" style="text-align: right;" cols="8" xl="3" lg="5" md="5" class="pt-0">
+                      <span  v-if="this.numberOfDuplicatePenRequests > 0"><span style="color: red;font-weight: bold;font-size: 1.2rem;">{{this.numberOfDuplicatePenRequests}}</span><span style="color: red;font-weight: normal;font-size: 1rem;"> prior PEN Requests</span></span>
```
Comment (mightycox) [inline frontend/src/components/PenRequestDetail.vue]: Vuetify has classes that you can use for styles: font-weight-bold red--text

File: frontend/src/components/PenRequestDetail.vue [hunk 3]
```diff
@@ -817,6 +824,22 @@ export default {
           this.loadingActionResults = false;
         });
     },
+    searchDuplicatePenRequestsByPen() {
+      this.loadingDuplicatePenRequests = true;
+      ApiService.apiAxios
+        .get(`${Routes.PEN_REQUEST_SEARCH_URL}duplicatePenRequests?pen=${this.penSearchId}`)
```
Comment (mightycox) [inline frontend/src/components/PenRequestDetail.vue]: The axios get function accepts params as an argument, so they don't have to be in the url string

File: frontend/src/components/PenRequestDetail.vue [hunk 4]
```diff
@@ -817,6 +824,22 @@ export default {
...
+        .then(response => {
+          console.log(response.data);
+          console.log(response.data.length);
```
Comment (mightycox) [inline frontend/src/components/PenRequestDetail.vue]: Should these be here?

File: frontend/src/components/PenRequestDetail.vue (hunk 2 above)
Comment (mishraomp) [reply frontend/src/components/PenRequestDetail.vue]: example please

**Pull request merged**